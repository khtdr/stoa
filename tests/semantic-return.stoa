#!/usr/bin/env stoa

{ // should't be allowed
  return;
}
