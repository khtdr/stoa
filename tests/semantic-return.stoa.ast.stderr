Parse error in ./semantic-return.stoa at line,col 4,3
4.   return;
     â†‘
No return from top-level allowed
